include/deal.II/base/function.h

```diff
@@ -649,10 +649,10 @@ class ComponentSelectFunction : public ConstantFunction<dim, RangeNumberType>
  *
  * The class gains additional expressive power because the argument it takes
  * does not have to be a pointer to an actual function. Rather, it is a
- * function object, i.e., it can also be the result of call to std::bind (or
- * boost::bind) or some other object that can be called with a single
- * argument. For example, if you need a Function object that returns the norm
- * of a point, you could write it like so:
+ * function object, i.e., it can also be the result of a call to a lambda
+ * function or some other object that can be called with a single argument. For
```

Comment:
Actually, it isn't a *call* to a lambda function, but the lambda function itself. That's different from what we had before where the function object is the result of a call to `std::bind`.